### PR TITLE
[SHEPHERD] Fix linter errors

### PR DESCRIPTION
**Pull request by brandonxxlee**
Update pylintrc, fix linter errors

### CODE REVIEW

File: runtime/Makefile [hunk 1]
```diff
@@ -9,7 +9,7 @@ artifacts-install:
 	$(nop)
 
 lint:
-	pylint Ansible.py runtime.py stateManager.py studentAPI.py
```
Comment (baby-bell) [inline runtime/Makefile]: ditto

File: hibike/travis/Makefile [hunk 2]
```diff
@@ -12,7 +12,7 @@ artifacts-install:
 	$(nop)
 
 lint:
-	cd .. && find . -name "*.py" | xargs pylint
```
Comment (baby-bell) [inline hibike/travis/Makefile]: I've already fixed the linter problems in #480, please don't disable them

**Comment by brandonxxlee** [comment]
I just wanted to merge this into shepherd/master, because we haven't merged in from master for a while. Maybe we should just merge in master into shepherd/master not sure.

**Comment by baby-bell** [comment]
My mistake. I should have read more closely. Just be careful when you merge into master.

**Review by baby-bell — APPROVED** [review]

**Pull request merged**